utilities/responses.py

```diff
@@ -51,102 +49,114 @@ def __call__(self, request):
 
         return response
 
+    @staticmethod
+    def __make_response(status_code: int, extra: dict):
+        return {
+            'success': True if status_code // 100 == 2 else False,
```

Comment:
It's not a clean code, other programmers my have difficulty to read this section

Reply:
which part? whole `__make_response` method or line 55?

Reply:
I think it's better to have a condition like 200 <= status_code < 300 it would be easier to understand the point of condition

Reply:
now we have a good approach